status/chat.nim

```diff
@@ -615,6 +620,16 @@ markAsReadProps: MarkAsReadNotificationProperties): string =
   
   self.events.emit("markNotificationsAsRead", markAsReadProps)
 
+proc markActivityCenterNotificationUnread*(self: ChatModel, notificationId: string,
+markAsUnreadProps: MarkAsUnreadNotificationProperties): string =
+  try:
+    status_chat.markActivityCenterNotificationsUnread(@[notificationId])
+  except Exception as e:
+    error "Error marking as unread", msg = e.msg
+    result = e.msg
+  
+  self.events.emit("markNotificationsAsUnread", markAsUnreadProps)
```

Comment:
should this still emit the event even if there's an exception?

Reply:
That's a valid question. Probably not. These changes are highly inspired by `markActivityCenterNotificationRead`